docker/env/.lido_terra.testnet.env

```diff
@@ -1,4 +1,4 @@
-SOURCE_ENDPOINTS=tequila-fcd.terra.dev,tequila-fcd.terra.dev
+SOURCE_ENDPOINTS=tequila-fcd.terra.dev,3.34.120.243:26657,167.99.25.150:26657
```

Comment:
we need a fcd/lcd nodes, seems like 3.34.120.243:26657,167.99.25.150:26657 are rpc nodes.

Reply:
Yeah, forgot to delete them